blog.md

```diff
@@ -0,0 +1,100 @@
+## Take a look at your repository from a new perspective with Git Machete Plugin!
+
```

Comment:
I'd consider smuggling the logo (preferably the extended version with `Git Machete` words) here at the beginning